kitsune/users/validators.py

```diff
@@ -0,0 +1,6 @@
+from django.core.validators import RegexValidator
+
+
+TwitterValidator = RegexValidator(r'^[\w]+$',
```

Comment:
I don't think this regex is right, because it doesn't allow numbers or underscores. You should add two tests for those cases and *then* edit this to make the tests pass.